### PR TITLE
Feat playlist

### PR DESCRIPTION
**Pull request by shubhamkumar648**
- add all feature of video library 
- 1. like video
- history video
- watch later video 
- playlist video page

### CODE REVIEW

File: src/page/singleVideoPage/SinglevideoPage.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      }
+    })();
+  }, [Id]);
+
+  return (
+    <div className="main-container">
+      <Sidebar />
+      <article>
+        {/* <SingleVideocard  singleVideo={singleVideo}  key={singleVideo._id}/>  */}
+
```
Comment (RahulSingh9131) [inline src/page/singleVideoPage/SinglevideoPage.js:31]: remove this if not needed.

**Review by RahulSingh9131 — APPROVED** [review]
with css app is looking more clean now. Code is also clean and understandable. PR is ready to merge.

File: src/App.css [hunk 2]
```diff
@@ -49,3 +47,52 @@
...
+/* sidebar css for all pages */
+
+.main-container {
+  display: grid;
+  grid-template-columns: 20% 80%;
+}
+
+.aside-content {
+  box-shadow: 0 0 27px 0 rgba(214, 231, 233, 0.52);
+  border: 1px solid rgba(0,0,0,.05);
```
Comment (aniketsingh13) [inline src/App.css]: use color variables

**Review by aniketsingh13 — APPROVED** [review]
code structure look good. code is easy to read good to merge

File: src/App.css (hunk 2 above)
Comment (shubhamkumar648) [reply src/App.css]: will do that

**Pull request merged**